src/gwt/src/org/rstudio/core/client/dom/DomUtils.java

```diff
@@ -895,4 +895,16 @@ public static final native void setSelectionRange(Element el, int start, int end
       if (el.setSelectionRange)
          el.setSelectionRange(start, end);
    }-*/;
+   
+   public static final native void copyCodeToClipboard(String text) /*-{
```

Comment:
Curious about browser compatibility here -- IIUC some browsers only allow you to call the `"copy"` command in certain contexts (ie in response to certain JS events)